docs/index.rst

```diff
@@ -145,3 +146,4 @@ Indices and tables
 * :ref:`modindex`
 * :ref:`search`
 
+THIS IS GREAT. CHEERS!
```

Comment:
While I really appreciate that kind of feedback as well, it would be a little easier to handle if it wasn't part of the content ;)